tools/cbmc/proofs/ARP/ARPGetCacheEntry/README.md

```diff
@@ -0,0 +1,4 @@
+The combined proofs in the subdirectories prove that ARPGetCacheEntry
+is memory safe for all possible combinations of ipconfigARP_STORES_REMOTE_ADDRESSES
+and ipconfigUSE_LLMNR. These are the only configuration
+parameters used inside the ARPGetCacheEntry.
```

Comment:
Is there any automation to detect the addition of configurations parameters to the function?

Reply:
Currently no. Agreed that this would be good to have.

On my todo list is a 'linter' for our proofs, which itself is run as a CI check, and which would fail when one of our proofs somehow goes out of date. This seems like a good feature to add to that---some sort of alarm that tells us that we're not checking as many configurations as we should be.